src/core/StatementFormatter.ts

```diff
@@ -19,34 +19,40 @@ export default class StatementFormatter {
   private query: WhitespaceBuilder;
 
   private currentNewline = true;
+  private inline = false;
   private previousReservedToken: Token = EOF_TOKEN;
   private previousCommandToken: Token = EOF_TOKEN;
-  private tokens: Token[] = [];
+  private nodes: AstNode[] = [];
   private index = -1;
 
-  constructor(cfg: FormatOptions, params: Params) {
+  constructor(cfg: FormatOptions, params: Params, { inline = false }: { inline?: boolean } = {}) {
```

Comment:
can we also extract flags param as a type

Reply:
Done in future PR.